### DISCUSSION TITLE
Fully qualified name missing from parent declarations

### DISCUSSION THREAD

**Discussion by JordanLongstaff** [LemonAppDev/konsist#1585] [Report a Problem] [answered]
### Tell us what version of Konsist you use.

0.17.0

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

These tests listed below are failing on a Renovate branch of my repo attempting to upgrade Konsist to version 0.17.0:

https://github.com/JordanLongstaff/ArtemisAgent/blob/renovate/konsist/IAN/annotations/konsist/src/test/kotlin/com/walkertribe/ian/AnnotationsKonsistTest.kt
https://github.com/JordanLongstaff/ArtemisAgent/blob/renovate/konsist/IAN/enums/konsist/src/test/kotlin/com/walkertribe/ian/enums/EnumsKonsistTest.kt
https://github.com/JordanLongstaff/ArtemisAgent/blob/renovate/konsist/IAN/packets/konsist/src/test/kotlin/com/walkertribe/ian/protocol/PacketKonsistTest.kt

They are failing because `fullyQualifiedName` was removed from some of the declaration types in PR #1234, and now many declaration types (namely, `KoClassDeclaration`, `KoInterfaceDeclaration` and `KoObjectDeclaration` for me) are missing them.

### Actual behaviour

Numerous instances of `Unresolved reference 'fullyQualifiedName'.`

### Expected behaviour

The tests should build and run as they did before.

**Comment by igorwojda** [comment]
Thx for report. We are looking into this.

**Comment by JordanLongstaff** [comment, marked as answer]
OK, I found the real cause. It's actually #1486 that changed things up on me. I think I can fix it myself now that I have a better understanding of the changes.
